.github/workflows/build.yml

```
@@ -47,9 +47,9 @@ jobs:
     name: ${{ matrix.os }} ${{ matrix.arch }} ${{ matrix.gcrypt }} ${{ matrix.compiler }} ${{ matrix.pcre }} ${{ matrix.maxminddb }} ${{ matrix.msan }}
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: false # Debugging purposes, switch to true when all issues on big endian is solved
+      fail-fast: true # Debugging purposes, switch to true when all issues on big endian is solved
```

```suggestion
      fail-fast: true
```

AFAIK the comment is not necessary anymore.

---

Not sure as we will have issues on windows for sure. So I will update it but I'll not remove it.